plugins/core/webview/src/q-ui/components/selectableItem.vue

```diff
@@ -20,7 +20,7 @@
                 />
             </svg>
             <svg
-                v-if="itemTitle == 'Workforce'"
+                v-if="itemId === LoginOption.ENTERPRISE_SSO"
```

Comment:
Enterprise string is differnt for Toolkit panel and Q panel, did we handle this?

Reply:
not yet

Reply:
so we say `workforce` in Toolkit and `professional license` in Q ?

Reply:
nm let me ask in our chat and take it as a followup

Reply:
>so we say workforce in Toolkit and professional license in Q ?
not sure for this part